docs/index.rst

```diff
@@ -1,359 +1,10 @@
-.. Kafka Connect JDBC documentation master file
```

Comment:
It will be helpful to have a reference to link to here like many other sections do.